src/main/java/frc/robot/flywheel/FlywheelSubsystem.java

```diff
@@ -13,35 +18,38 @@ public class FlywheelSubsystem extends SubsystemBase {
      * 1 Follower TalonFX (Right)
      * Total: 2 TalonFX motors
      */
+    TalonFX left = new TalonFX(FlywheelConstants.LEFT_CAN_ID);
```

Comment:
You should be creating the hardware in the constructor in case the hardware hasn't initialized. When the constructor is run we know the hardware is ready. You should also make these private to make sure nothing else can access it.

Reply:
I would also add which motor is the master in the variable name.
Something like `leftMasterMotor`.